dracut/anaconda-hmcroot

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# anaconda-hmcroot:
+# Provide access to the root on the USB/DVD drive
+# of the SE/HMC of System z hardware.
+
+command -v getargbool >/dev/null || . /lib/dracut-lib.sh
+command -v config_get >/dev/null || . /lib/anaconda-lib.sh
```

Comment:
If I understand this `command -v` correctly it is testing command availability if so why do you testing these commands if they are not used here anywhere?

Reply:
Fixed, but you test the availability of a library, so it doesn't matter what command you test.

Reply:
It depends, you can for example rename the command or deprecate it, so there should be commands which you want to use to have valid feedback.